package.json

```diff
@@ -11,16 +11,19 @@
   },
   "bugs": "https://github.com/kitematic/kitematic/issues",
   "scripts": {
+    "build":"npm run tslint && tsc",
     "start": "grunt",
     "start-dev": "NODE_ENV=development grunt",
     "test": "jest -c jest-unit.json",
     "integration": "jest -c jest-integration.json",
+    "postinstall": "npm run build",
     "release": "grunt release",
     "release-verbose": "grunt --verbose release",
-    "lint": "eslint src"
+    "lint": "eslint src",
+    "tslint":"tslint --fix -p tsconfig.json"
   },
   "license": "Apache-2.0",
-  "electron-version": "0.35.4",
+  "electron-version": "1.7.10",
```

Comment:
Can you confirm that you  can still build and launch this on osx/windows? last update to Electron caused some issues for me.

Reply:
Yes, it is working well, but in not sure if this field is required now